.github/workflows/build-and-test.yml

```diff
@@ -3157,43 +3177,100 @@ jobs:
     runs-on: ubuntu-20.04
     if: ${{ github.event_name == 'pull_request' }}
     strategy:
+      fail-fast: false
       matrix:
-        arch: [ "armv7", "aarch64" ]
+        include:
+          - arch: "armv7"
+            system: "buster"
+            multistrap-config: "multistrap_raspbian_buster.conf"
+          - arch: "armv7"
+            system: "bullseye"
+            multistrap-config: "multistrap_raspbian_bullseye.conf"
+          - arch: "aarch64"
+            system: "buster"
+            multistrap-config: "multistrap_armbian64_buster.conf"
     env:
       CI_TMP_DIR: ${{ github.workspace }}/tmp
       DEBIAN_FRONTEND: "noninteractive"
-      SYSTEM_RASPBIAN: ${{ github.workspace }}/chroot-${{ matrix.arch }}
+      SYSTEM_RASPBIAN: ${{ github.workspace }}/chroot-${{ matrix.arch }}-${{ matrix.system }}
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
       - name: "Install and setup chroot"
         uses: ./.github/actions/multistrap
         with:
-          arch: ${{ matrix.arch }}
+          multistrap-config: ${{ matrix.multistrap-config }}
           packages: "bash wget curl sox xxd libatlas3-base libopenblas-base ca-certificates python3 python3-pip gnupg libatk1.0-0 libatk-bridge2.0-0 libcairo2 libcups2 libdbus-1-3 libgdk-pixbuf2.0-0 libgtk-3-0 libgbm1 libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libx11-xcb1 libxcb-dri3-0 libxcomposite1 libxcursor1 libxdamage1 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 xauth"
       - name: "Create a chroot tarball"
         run: |
           sudo tar -cf - -C ${{ env.SYSTEM_RASPBIAN }}/ --one-file-system . | xz -9 -T0 > ${{ github.workspace }}/chroot.tar.xz
       - uses: actions/upload-artifact@v2
         with:
-          name: chroot-${{ matrix.arch }}
+          name: chroot-${{ matrix.arch }}-${{ matrix.system }}
           path: ${{ github.workspace }}/chroot.tar.xz
   test-cpp-LinuxArm:
     name: "LinArm*|Test C++ binary"
     runs-on: ubuntu-20.04
     needs: [ build-lib-LinuxArmv7, build-lib-LinuxAarch64, train-test-model-Linux, build-test-chroot ]
     if: ${{ github.event_name == 'pull_request' }}
     strategy:
+      fail-fast: false
       matrix:
-        arch: [ "armv7", "aarch64" ]
-        models: ["test", "prod"]
-        samplerate: ["8000", "16000"]
+        include:
```

Comment:
what does it changes ?

Reply:
@lissyx There was a new `include` parameter added (`system`), please see this example:

```
- arch: "armv7"
  system: "buster"
  models: "test"
  samplerate: "8000"
```

FWIW, this was added by @reuben before I took over and resurrected the PR :)

Reply:
ok, nevermind then